google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -106,7 +107,8 @@
 
   private static final PathTemplate PROJECT_NAME_TEMPLATE =
       PathTemplate.create("projects/{project}");
-
+  private static final int MAX_MESSAGE_SIZE = 100 * 1024 * 1024;
+  
```

Comment:
Correct me if i am wrong since I don't have enough context.

This implementation (GapicSpannerRpc) is not currently being used (even on this dev branch?). When replacing SpannerRpc could you also:

1. add some IntegrationTests (for correctness & performance) here - google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/...
2. Sync from the public repo - since we may have added new GRPC calls

Reply:
1. GapicSpannerRpc is used in dev branch but not in master. see https://github.com/GoogleCloudPlatform/google-cloud-java/blob/spanner-gapic-migration/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java#L162.

2. I'll sync from master when this part of set-ups is done (watchdog, channelpool, logging). The repo has been refactored so I need to do that anyway.